fvm/evm/stdlib/contract.cdc

```diff
@@ -160,14 +219,14 @@ contract EVM {
             data: [UInt8],
             gasLimit: UInt64,
             value: Balance
-        ): [UInt8] {
-             return InternalEVM.call(
-                 from: self.addressBytes,
-                 to: to.bytes,
-                 data: data,
-                 gasLimit: gasLimit,
-                 value: value.attoflow
-            )
+        ): Result {
+            return InternalEVM.call(
+                from: self.addressBytes,
+                to: to.bytes,
+                data: data,
+                gasLimit: gasLimit,
+                value: value.attoflow
+            ) as! Result
```

Comment:
Why is the force-cast required here? Can the return type of `InternalEVM.call` be `Result`?

Reply:
Not possible, added a comment